examples/geocoding/components/geocoder-example/geocoder-example.tsx

```diff
@@ -1,83 +1,92 @@
 import {useEffect, useState} from 'react';
 import {useGeocoder, useGoogleMap} from '@ubilabs/google-maps-react-hooks';
 
+const initialPosition = {lat: 51.08998021141488, lng: 10.627828045134935};
+
 const GeocoderExample = () => {
   const map = useGoogleMap();
 
   // Get the geocoder from the useGeocoder hook
   const geocoder = useGeocoder();
 
-  const initialPosition = {lat: 51.08998021141488, lng: 10.627828045134935};
-
   const [marker, setMarker] = useState<google.maps.Marker | null>(null);
   const [infoWindow, setInfoWindow] = useState<google.maps.InfoWindow | null>(
     null
   );
 
+  // Add marker and info window to the map
   useEffect(() => {
     if (!map) {
       return () => {};
     }
 
     // Add an initial marker
-    if (!marker) {
-      const initialMarker = new google.maps.Marker({
-        map,
-        position: initialPosition
-      });
-
-      setMarker(initialMarker);
+    const initialMarker = new google.maps.Marker({
+      map,
+      position: initialPosition
+    });
 
-      return () => {};
-    }
+    setMarker(initialMarker);
 
     // Add an initial infowindow
-    if (!infoWindow) {
-      const initialInfoWindow = new google.maps.InfoWindow({
-        content:
-          'Click somewhere on the map to reverse geocode the position to an address.',
-        position: initialPosition
-      });
+    const initialInfoWindow = new google.maps.InfoWindow({
```

Comment:
Maybe we rename it to `newInfoWindow` and `newMarker`(above). Just thinking that we actually do not ever change the infowindow or marker, we just update the position and content. Just a thought. What do you think?